bsk_rl/envs/general_satellite_tasking/simulation/dynamics.py

```diff
@@ -733,7 +739,13 @@ def _set_storage_unit(
     @aliveness_checker
     def data_storage_valid(self) -> bool:
         """Check that the buffer has not run out of space."""
-        return self.storage_level <= self.storageUnit.storageCapacity
+        storage_check = self.storageUnitValidCheck
+        if storage_check or None:
```

Comment:
Does `or None` do anything here?

Reply:
No, I forgot to remove it. Now it should be fine.